lib_ecstasy/src/main/x/ecstasy.x

```diff
@@ -119,12 +119,19 @@ module ecstasy.xtclang.org {
             extends IllegalState(text, cause);
 
     /**
-     * An `UnsupportedOperation` exception is raised when an attempt is made to invoke functionality
-     * that is not present or has not yet been implemented.
+     * An `Unsupported` exception is raised when an attempt is made to invoke functionality that is
+     * not present, has not yet been implemented, or is otherwise not available.
      */
-    const UnsupportedOperation(String? text = Null, Exception? cause = Null)
+    const Unsupported(String? text = Null, Exception? cause = Null)
             extends Exception(text, cause);
 
+    /**
+     * A `NotImplemented` exception is raised when a `TODO` is encountered or an `assert:TODO`
```

Comment:
Maybe the doc should use "T0D0" (zeros instead of Os) to avoid the unnecessary TODO highlight, since this is not a TODO